Packages/PrimitivesComponents/Sources/Errors/RequiredFieldError.swift

```diff
@@ -0,0 +1,17 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+import Localization
+
+public enum RequiredFieldError: Equatable {
+    case field(name: String)
+}
+
+extension RequiredFieldError: LocalizedError {
+    public var errorDescription: String? {
+        guard case .field(let name) = self else {
```

Comment:
If we decide to use an enum, I suggest using a switch statement. This way, when a new case is added, the developer won't forget to handle it